src/Type/MediaItem/MediaItemType.php

```diff
@@ -121,9 +121,12 @@ public static function fields( $fields ) {
 				'description' => __( 'Details about the mediaItem', 'wp-graphql' ),
 				'resolve'     => function ( \WP_Post $post, $args, $context, ResolveInfo $info ) {
 					$media_details       = wp_get_attachment_metadata( $post->ID );
-					$media_details['ID'] = $post->ID;
+					if(!empty($media_details)) {
```

Comment:
let's adjust the spacing here: 

```suggestion
					if ( ! empty( $media_details ) ) {
```